vector/src/main/res/menu/menu_devtools.xml

```diff
@@ -9,6 +9,7 @@
         tools:visible="true"
         app:showAsAction="ifRoom"
         android:icon="@drawable/ic_edit"
+        app:iconTint="?vctr_content_secondary"
```

Comment:
Thanks for also having fixed that 👍 
<img width="287" alt="image" src="https://user-images.githubusercontent.com/3940906/140166070-4d14a4c2-1bf3-4756-9996-d91783c00b93.png">